modules/400-descheduler/templates/deployment.yaml

```diff
@@ -1,49 +1,51 @@
-{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+{{- range $d := .Values.descheduler.internal.deschedulers }}
+  {{- $deschedulerSpec := $d.spec }}
+  {{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
 ---
 apiVersion: autoscaling.k8s.io/v1
 kind: VerticalPodAutoscaler
 metadata:
-  name: descheduler
+  name: descheduler-{{ $d.metadata.name }}
   namespace: d8-descheduler
   {{- include "helm_lib_module_labels" (list . (dict "app" "descheduler")) | nindent 2 }}
 spec:
   targetRef:
     apiVersion: "apps/v1"
     kind: Deployment
-    name: descheduler
+    name: descheduler-{{ $d.metadata.name }}
   updatePolicy:
     updateMode: "Auto"
```

Comment:
I saw VPA options in the CRD, but they aren't used in templates.

Reply:
VPA removed.